app/assets/javascripts/resources.js

```diff
@@ -29,6 +29,55 @@ function scrollToTop(event) {
   $('html, body').animate({scrollTop: 0}, 'slow');
 }
 
+function getCookie(cname) {
```

Comment:
Please add a comment here saying that this code is copied from W3 Schools, with the link: http://www.w3schools.com/js/js_cookies.asp